pkg/models/module.go

```diff
@@ -0,0 +1,28 @@
+package models
+
+// Module is an interface to describe the domain model of a building block. A Module comes both
+// from an application developer-centric schema and workspace config, which can fully describes a workload,
+// an accessory, etc., such as database.Database, network.Port.
+// A Module can be built into a or a set of Resource, which will be consumed by Kusion engine.
+type Module interface {
+	// ModuleName returns the name to identify the module uniquely.
+	ModuleName() string
```

Comment:
Please rename `ModuleName` to `Name` to follow the Go convention

Reply:
The function `ModuleName` is used to define a module. If changed to `Name`, the capacity of definition will not exist.